docs/courses/creating-courses.md

```diff
@@ -0,0 +1,19 @@
+## Creating a new LibreLingo course
+
+Beginners should not be bothered by this page. This page is for those who want to create new courses from scratch, which is for those who have sufficient understanding of the project and how the courses work. Please continue at your own discretion.
+
+---
+
+**WARNING**: If you just want to make any existing course to be available on your language, please refer to [this page](editing-courses.md) instead and ignore this page. Following this page for editing existing courses can cause invalid data to be created.
```

Comment:
This is about translating an existing course to a different source language? 

I think we can leave this out for now, because this is a little bit more complicated. The _structure_ of a course can be different based on the source language *and* the target language

Reply:
I included this in case someone comes to this page by mistake and tries to follow it for making a translation. This is just to guide them to the right place.

Reply:
> Following this page for editing existing courses can cause invalid data to be created.

no sure what this part means

> If you just want to make any existing course to be available on your language

It's just not exactly clear to me what that even means. I would leave this out because this is really a more complicated topic imo

Reply:
> Following this page for editing existing courses can cause invalid data to be created.

I think I followed the current instructions on this page to create a new skill `Id` (UUID v4) for each one and it messed up the system. It had to be reverted to previous values. When translating Spanish course, I thought I was creating "new" course, but actually the docs meant I was not. I wanted to point that out so that people who are translating courses do not fall for this and do what I did.

How about this...?

```suggestion
**NOTE**: If you just want to edit existing courses or translate one, try [this page about editing courses](editing-courses.md) instead.
```

If someone else makes this mistake, we can add something to clear the confusion.